content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -363,6 +366,44 @@ will be terminated once the job backoff limit has been reached. This can make de
 from failed Jobs is not lost inadvertently.
 {{< /note >}}
 
+### Backoff limit per index {#backoff-limit-per-index}
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+{{< note >}}
+You can only configure the backoff limit per index for a Job, if you
+have the `JobBackoffLimitPerIndex` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+enabled in your cluster. A per-index backoff limit is also only available for indexed Jobs that
+use the `restartPolicy: Never`.
+{{< /note >}}
+
+When you run an [indexed](#completion-mode) Job, you may want to handle pod failures for different
+indexes as counted independently. To do so, set the `.spec.backoffLimitPerIndex`
+to specify the the maximal number of pod failures per index. For example, to allow exactly
+one retry for each indexed subtask in a Job and to fail early if the retry also fails, set `.spec.backoffLimitPerIndex` to 1 and set `.spec.maxFailedIndexes` to 0.
```

Comment:
This is too specific at this point. Maybe you can have a paragraph at the end of the section saying:

```
Here are some sample configurations you can set for your indexed Jobs:
- `backoffLimitPerIndex=1` and `maxFailedIndexes=0`, meaning that...
```

Reply:
I think this is something we could consider moving to the tasks page. I'm thinking about "Execute all indexes" title which would correspond with the original Issue: https://github.com/kubernetes/kubernetes/issues/109712.

Reply:
Resolving as the original comment is addressed, and the need to have a new tasks page is discussed in a separate thread: https://github.com/kubernetes/website/pull/41921#discussion_r1272202923